### ISSUE TITLE
Upload File Tab not avilable in Device Import to Upload CSV file

### ISSUE THREAD

**Issue by parkerschoenig** [netbox-community/netbox#11528]
### NetBox version

v3.4.1

### Python version

3.8

### Steps to Reproduce

Creating based on Q&A discussion item #11370. I see an issue was not opened for this. This feature is useful for other users in our instance since we use a template with predefined field options / custom field options in the import file. 

On the Devices Import page, the option to import from a CSV file is missing.

![image](https://user-images.githubusercontent.com/56207887/213218501-9c8cf9de-14cd-44b9-8acf-977d6fef7585.png)

For example, on IP Address import that option is present:

![image](https://user-images.githubusercontent.com/56207887/213218960-3e7d0c09-d8a2-4633-9f1a-fcb25ce4fdf1.png)

### Expected Behavior

Import tab is available on the Devices Import page to import a CSV file. 

### Observed Behavior

The import tab is missing.

**Comment by proy-rbbn** [comment]
Could we expand this to make sure we get options to upload "racked devices" as well as "child_devices"?

**Comment by jeremystretch** [comment]
I took a brief look at this earlier today. Ideally I'd like to remove the "child devices" tab altogether but it wasn't immediately apparent whether it's still necessary to maintain a separate view & form to import child devices specifically.

**Comment by proy-rbbn** [comment]
yeah - I would prefer 1 form to handle either child or parent... It is interesting that when adding a child device manually you do not have to enter a host device or device bay.... so it is inconsitent already....

**Comment by proy-rbbn** [comment]
Thank-you so much for adding device CSV file upload. I just tested device import vua Upload File in v3.4.5 and found that the 1st field in the import is always skipped.
I worked around it by adding a blank field to the CSV file as below. Would be nice to fix this bug.
e.g. this works.
,name,device_role,manufacturer,device_type,status,site,location,rack,position,face
,Arcot,Server,Sun Microsystems,Netra T5220,active,Bangalore,BLR-Row X107,X107Y01,12,front

e.g. the below imports successfully, but the device name is not populated.
name,device_role,manufacturer,device_type,status,site,location,rack,position,face
Arcot,Server,Sun Microsystems,Netra T5220,active,Bangalore,BLR-Row X107,X107Y01,12,front
